sig-instrumentation/annual-report-2024.md

```diff
@@ -30,54 +44,54 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
-<!-- 
-  - Alpha
-    - [4827 - Component Statusz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4827-component-statusz) - v1.32
-    - [4828 - Component Flagz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4828-component-flagz) - v1.32
+- Alpha
+  - [4827 - Component Statusz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4827-component-statusz) - v1.32
+  - [4828 - Component Flagz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4828-component-flagz) - v1.32
 
-  - Beta
-    - [3077 - Contextual logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging) - v1.30
+- Beta
+  - [3077 - Contextual logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging) - v1.30
 
-  - Stable
-    - [2305 - Dynamic Cardinality Enforcement](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2305-metrics-cardinality-enforcement) - v1.30
-    - [647 - APIServer Tracing](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/647-apiserver-tracing) - v1.30 -->
+- Stable
+  - [2305 - Dynamic Cardinality Enforcement](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2305-metrics-cardinality-enforcement) - v1.30
 
 ## [Subprojects](https://git.k8s.io/community/sig-instrumentation#subprojects)
 
+**New in 2024:**
+
+- [zpages](https://git.k8s.io/kubernetes/staging/src/k8s.io/component-base/zpages)
```

Comment:
```suggestion
- [zpages](https://git.k8s.io/kubernetes/staging/src/k8s.io/component-base/zpages)
- [usage-metrics-collector](https://github.com/kubernetes-sigs/usage-metrics-collector) implemented `cgroupv2` support in the metrics sampler. Additionally, it had 6 gained new contributors from 4 different companies.
```

Putting this down here for a "fast-commit" in case we miss Elana before the deadline.

Reply:
Um, hadnt we introduced usage-metrics-collector in 2023, [ref](https://github.com/kubernetes/community/blob/master/sig-instrumentation/annual-report-2023.md?plain=1#L57)

I put the recent development around the cgroupv2 support under `What work did the SIG do this year that should be highlighted?`

Reply:
Ah, right.